static/includes/TNDataCollection.md

```diff
@@ -21,3 +22,18 @@ This table describes the differences:
 |              | User-created System and dataset names | Software versions, firmware versions |
 |              | Directory, files names, user data | Services and features enabled, Usage and Performance statistics |
 {{< /truetable >}}
+
+{{< expand "Reviewing Usage Collection" "v" >}}
+To review system usage data collected by TrueNAS, access the system shell via SSH or the console and enter `midclt call usage.gather`.
+This command returns the same statistics that are transmitted to iXsystems when **Usage collection** is enabled.
+{{< /expand >}}
+
+TrueNAS does not collect any user-defined names for servers, pools, datasets, shares, files, applications/jails, processes, and similar.
+In TrueNAS SCALE, applications do transmit chart release names for deployed applications (e.g. “minio/2.0.6”) but not user-defined names.
+
+{{< hint type=important title="Debug files do contain sensitive user data" >}}
+While **Usage collection** collects only non-sensitive system data, sensitive user data is included in a TrueNAS-generated [debug file](https://www.truenas.com/docs/contributing/issuereporting/jiraissuereporting/#downloading-a-debug-file), such as such as those requested for a bug report, feature request, or other troubleshooting.
```

Comment:
We are talking about usage collection in general here and not the UI element so we do not need the emphasis.
```suggestion
While usage collection gathers only non-sensitive system data, sensitive user data is included in a TrueNAS-generated [debug file](https://www.truenas.com/docs/contributing/issuereporting/jiraissuereporting/#downloading-a-debug-file), such as those requested for a bug report, feature request, or other troubleshooting.
```